changelog/PR7356.dd

```diff
@@ -0,0 +1,40 @@
+`opDispatch` resolution in `with` statements
+
+`opDispatch` resolution in `with` statements has been modified to bring it into compliance with the existing scope resolution rules in the language specification.  See $(LINK2 $(ROOT_DIR)spec/statement.html#with-statement, WithStatement) in the language specification; specifically 11.19.6.
```

Comment:
Until we come up with a good system, please refrain from using hardcoded numeric chapter and verse.